sympy/physics/vector/tests/test_frame.py

```diff
@@ -184,6 +178,18 @@ def test_w_diff_dcm():
            (c11*c12d + c21*c22d + c31*c32d)*b.z)
     assert b.ang_vel_in(a) - expr == 0
 
+def test_w_diff_dcm2():
+    q1, q2, q3 = dynamicsymbols('q1:4')
+    N = ReferenceFrame('N')
+    A = N.orientnew('A', 'axis', [q1, N.x])
+    B = A.orientnew('B', 'axis', [q2, A.y])
+    C = B.orientnew('C', 'axis', [q3, B.z])
+
+    DCM = C.dcm(N).T
+    D = N.orientnew('D', 'DCM', DCM)
+
+    assert D.dcm(N) == C.dcm(N)
```

Comment:
Can you also include explicitly what this is testing for rather than just comparing the output of two different functions? 

Otherwise this test will pass even if both just output e.g. zero.

Reply:
I have added some documentation for both test functions and rewritten the original ```test_w_diff_dcm1()``` since it looks not good.